v2/testing/validate/json-validation.js

```diff
@@ -23,7 +23,8 @@ _.each(args.projects, (projectAdapter) => {
     });
 
     it('category matches one of the defined categories', () => {
-      chai.expect(projectAdapter.category).to.be.oneOf(categories);
+      chai.expect(projectAdapter.category,
+        `Valid categories: ${categories.join(',')}`).to.be.oneOf(categories);
```

Comment:
Great to see we have some tests in there.
Would it make sense to create a GitHub Actions pipeline later so we can catch it next time it fail?

Reply:
I was totally planning on doing that soon! Others are failing.